keps/sig-multicluster/2149-clusterid/README.md

```diff
@@ -341,68 +419,134 @@ Contains a unique identifier for the containing cluster.
-may be composed of two DNS labels separated with a dot. The additional DNS label
-is implementation-dependent and allows the implementation to uniquely identify
-the cluster registry with which a cluster is registered.
+within the clusterset. In such case the value of `cluster.clusterset.k8s.io
+ClusterProperty` may be composed of two DNS labels separated with a dot. The
+additional DNS label is implementation-dependent and allows the implementation
+to uniquely identify the cluster registry with which a cluster is registered.
 
 #### Property: `clusterset.k8s.io`
 
-Contains an identifier that relates the containing cluster to the ClusterSet in which it belongs.
+Contains an identifier that relates the containing cluster to the ClusterSet in
+which it belongs.
 
 
 ##### Lifespan
 
-*   The identifier **must** exist and be immutable for the duration of a cluster’s membership in a ClusterSet.
-*   The identifier **must not** exist when the cluster is not a member of a ClusterSet.
+*   The identifier **must** exist and be immutable for the duration of a
+    cluster’s membership in a ClusterSet.
+*   The identifier **must not** exist when the cluster is not a member of a
+    ClusterSet.
 
 
 ##### Contents
 
 *   The identifier **must** associate the cluster with a ClusterSet.
-*   The identifier **may** be either unique or shared by members of a ClusterSet.
+*   The identifier **may** be either unique or shared by members of a
+    ClusterSet.
 
 
 ##### Consumers
 
-*   **Must** be able to rely on the identifier existing, unmodified for the entire duration of its membership in a ClusterSet.
-*   **Should** watch the clusterset property to detect the span of a cluster’s membership in a ClusterSet.
+*   **Must** be able to rely on the identifier existing, unmodified for the
+    entire duration of its membership in a ClusterSet.
+*   **Should** watch the clusterset property to detect the span of a cluster’s
+    membership in a ClusterSet.
 
 
 ### Additional Properties
 
-Implementers are free to add additional properties as they see fit, so long as they do not conflict with the well known properties _and_ utilize a suffix. The following suffixes are reserved for Kubernetes and related projects: `.k8s.io`, `.kubernetes.io`. For example, an implementation may utilize the `Kind` `ClusterProperty` to store objects with the name `fingerprint.coolmcsimplementation.com` but not `fingerprint.k8s.io` and not simply `fingerprint`.
-
+Implementers are free to add additional properties as they see fit, so long as
+they do not conflict with the well known properties _and_ utilize a suffix. The
+following suffixes are reserved for Kubernetes and related projects: `.k8s.io`,
+`.kubernetes.io`. For example, an implementation may utilize the `Kind`
+`ClusterProperty` to store objects with the name
+`fingerprint.coolmcsimplementation.com` but not `fingerprint.k8s.io` and not
+simply `fingerprint`.
```

Comment:
We usually reserve un-spaced names for cluster operators - any reason not to do so here?